roachtest: activerecord failed [relation does exist + others]

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18676807?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18676807?buildTab=artifacts#/activerecord) on master @ [5b931856d005214057f0703890ce3a77cdc21f30](https://github.com/cockroachdb/cockroach/commits/5b931856d005214057f0703890ce3a77cdc21f30):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-5b931856d005214057f0703890ce3a77cdc21f30
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18676807/activerecord/1738305330660/1738313343960)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47032

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18692235?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18692235?buildTab=artifacts#/activerecord) on master @ [f7d019d5e675fa2711b61ffd54a9990b6a1a1da8](https://github.com/cockroachdb/cockroach/commits/f7d019d5e675fa2711b61ffd54a9990b6a1a1da8):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-f7d019d5e675fa2711b61ffd54a9990b6a1a1da8
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18692235/activerecord/1738391668512/1738400766365)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18700140?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18700140?buildTab=artifacts#/activerecord) on master @ [507c68f8abb0c9a3f527c66cb42129c2cb096e7d](https://github.com/cockroachdb/cockroach/commits/507c68f8abb0c9a3f527c66cb42129c2cb096e7d):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-507c68f8abb0c9a3f527c66cb42129c2cb096e7d
Tests run against activerecord 7.2.1
8478 Total Tests Run
8473 tests passed
5 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18700140/activerecord/1738478076801/1738486933036)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18705847?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18705847?buildTab=artifacts#/activerecord) on master @ [507c68f8abb0c9a3f527c66cb42129c2cb096e7d](https://github.com/cockroachdb/cockroach/commits/507c68f8abb0c9a3f527c66cb42129c2cb096e7d):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-507c68f8abb0c9a3f527c66cb42129c2cb096e7d
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18705847/activerecord/1738564460170/1738574607510)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18718899?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18718899?buildTab=artifacts#/activerecord) on master @ [82b1a2b174dcedfd23b0a3a40c6069143f4b038f](https://github.com/cockroachdb/cockroach/commits/82b1a2b174dcedfd23b0a3a40c6069143f4b038f):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-82b1a2b174dcedfd23b0a3a40c6069143f4b038f
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18718899/activerecord/1738650906907/1738658994883)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18731442?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18731442?buildTab=artifacts#/activerecord) on master @ [2e7d5a6d0194411d0f91fe4f3eba23fcea63ccac](https://github.com/cockroachdb/cockroach/commits/2e7d5a6d0194411d0f91fe4f3eba23fcea63ccac):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-2e7d5a6d0194411d0f91fe4f3eba23fcea63ccac
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18731442/activerecord/1738737214912/1738744370621)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18744720?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18744720?buildTab=artifacts#/activerecord) on master @ [bce8eb6edbbe879d376ab0c99e9d1129e5afef96](https://github.com/cockroachdb/cockroach/commits/bce8eb6edbbe879d376ab0c99e9d1129e5afef96):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-bce8eb6edbbe879d376ab0c99e9d1129e5afef96
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18744720/activerecord/1738823621039/1738831800458)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18757779?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18757779?buildTab=artifacts#/activerecord) on master @ [13d8c5191614530b50802f77081f15d2ce2ca705](https://github.com/cockroachdb/cockroach/commits/13d8c5191614530b50802f77081f15d2ce2ca705):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-13d8c5191614530b50802f77081f15d2ce2ca705
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18757779/activerecord/1738910136557/1738918999057)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18769596?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18769596?buildTab=artifacts#/activerecord) on master @ [bbaa2e50b2fe789527aac09b99fa5eee432e7695](https://github.com/cockroachdb/cockroach/commits/bbaa2e50b2fe789527aac09b99fa5eee432e7695):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-bbaa2e50b2fe789527aac09b99fa5eee432e7695
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/cpu_arch=arm64/run_1
```

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18769596/activerecord/1738996304827/1739007929465)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1.0-rc release-blocker]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18775094?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18775094?buildTab=artifacts#/activerecord) on master @ [ecfb74b7ad75327f11814966a2cdab1b9793a549](https://github.com/cockroachdb/cockroach/commits/ecfb74b7ad75327f11814966a2cdab1b9793a549):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-ecfb74b7ad75327f11814966a2cdab1b9793a549
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18775094/activerecord/1739082864008/1739089894070)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18811231?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18811231?buildTab=artifacts#/activerecord) on master @ [007029617564b747e4f199c990860ad73f299c3a](https://github.com/cockroachdb/cockroach/commits/007029617564b747e4f199c990860ad73f299c3a):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-007029617564b747e4f199c990860ad73f299c3a
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18811231/activerecord/1739342147533/1739349150071)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18821720?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18821720?buildTab=artifacts#/activerecord) on master @ [95982118802e223479d9d2c9a3dcf1c825e7c647](https://github.com/cockroachdb/cockroach/commits/95982118802e223479d9d2c9a3dcf1c825e7c647):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-95982118802e223479d9d2c9a3dcf1c825e7c647
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18821720/activerecord/1739428275362/1739435358151)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18841856?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18841856?buildTab=artifacts#/activerecord) on master @ [67a82bfc09766413d7f8ae8edb10035940b94c7e](https://github.com/cockroachdb/cockroach/commits/67a82bfc09766413d7f8ae8edb10035940b94c7e):


```
Tests run against activerecord 7.2.1
8477 Total Tests Run
8442 tests passed
35 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
34 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_select_with_strings (unexpected)
--- FAIL: PostgresqlArrayTest#test_quoting_non_standard_delimiters (unexpected)
--- FAIL: PostgresqlArrayTest#test_not_compatible_with_serialize_array (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_column (unexpected)
--- FAIL: PostgresqlArrayTest#test_assigning_non_array_value (unexpected)
--- FAIL: PostgresqlArrayTest#test_strings_with_commas (unexpected)
--- FAIL: PostgresqlArrayTest#test_precision_is_respected_on_timestamp_columns (unexpected)
--- FAIL: PostgresqlArrayTest#test_assigning_empty_string (unexpected)
--- FAIL: PostgresqlArrayTest#test_change_column_cant_make_non_array_column_to_array (unexpected)
--- FAIL: PostgresqlArrayTest#test_insert_fixture (unexpected)
--- FAIL: PostgresqlArrayTest#test_assigning_valid_pg_array_literal (unexpected)
--- FAIL: PostgresqlArrayTest#test_type_cast_array (unexpected)
--- FAIL: PostgresqlArrayTest#test_attribute_for_inspect_for_array_field (unexpected)
--- FAIL: PostgresqlArrayTest#test_rewrite_with_integers (unexpected)
--- FAIL: PostgresqlArrayTest#test_mutate_array (unexpected)
--- FAIL: PostgresqlArrayTest#test_type_cast_integers (unexpected)
--- FAIL: PostgresqlArrayTest#test_select_with_integers (unexpected)
--- FAIL: PostgresqlArrayTest#test_with_empty_strings (unexpected)
--- FAIL: PostgresqlArrayTest#test_datetime_with_timezone_awareness (unexpected)
--- FAIL: PostgresqlArrayTest#test_strings_with_null_strings (unexpected)
--- FAIL: PostgresqlArrayTest#test_contains_nils (unexpected)
--- FAIL: PostgresqlArrayTest#test_strings_with_array_delimiters (unexpected)
--- FAIL: PostgresqlArrayTest#test_escaping (unexpected)
--- FAIL: PostgresqlArrayTest#test_strings_with_quotes (unexpected)
--- FAIL: PostgresqlArrayTest#test_string_quoting_rules_match_pg_behavior (unexpected)
--- FAIL: PostgresqlArrayTest#test_encoding_arrays_of_utf8_strings (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- FAIL: PostgresqlArrayTest#test_where_by_attribute_with_array (unexpected)
--- FAIL: PostgresqlArrayTest#test_default (unexpected)
--- FAIL: PostgresqlArrayTest#test_rewrite_with_strings (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_default_strings (unexpected)
--- FAIL: PostgresqlArrayTest#test_attribute_for_inspect_for_array_field_for_large_array (unexpected)
--- FAIL: PostgresqlArrayTest#test_array_with_serialized_attributes (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/cpu_arch=arm64/run_1
```

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=false</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18841856/activerecord/1739601347409/1739611401237)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18844760?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18844760?buildTab=artifacts#/activerecord) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-34e34c2d5b9598441b935f03a794277e9329a256
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18844760/activerecord/1739687616652/1739694249382)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18847262?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18847262?buildTab=artifacts#/activerecord) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-34e34c2d5b9598441b935f03a794277e9329a256
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18847262/activerecord/1739773859405/1739783169111)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18851605?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18851605?buildTab=artifacts#/activerecord) on master @ [f3193fa36f11d583dd7e7ba505a89e7e309996aa](https://github.com/cockroachdb/cockroach/commits/f3193fa36f11d583dd7e7ba505a89e7e309996aa):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-f3193fa36f11d583dd7e7ba505a89e7e309996aa
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18851605/activerecord/1739860223032/1739867532423)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18863967?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18863967?buildTab=artifacts#/activerecord) on master @ [954761451e68c1c5db5fbe0f336225955dad1075](https://github.com/cockroachdb/cockroach/commits/954761451e68c1c5db5fbe0f336225955dad1075):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-954761451e68c1c5db5fbe0f336225955dad1075
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18863967/activerecord/1739946948446/1739955946765)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18875921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18875921?buildTab=artifacts#/activerecord) on master @ [3e38c3357dc026e41cf48d192611608b34fc064a](https://github.com/cockroachdb/cockroach/commits/3e38c3357dc026e41cf48d192611608b34fc064a):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-3e38c3357dc026e41cf48d192611608b34fc064a
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18875921/activerecord/1740033386338/1740041171858)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18887368?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18887368?buildTab=artifacts#/activerecord) on master @ [6bada5e7bc53457d56fc92acac8cac29127acbcb](https://github.com/cockroachdb/cockroach/commits/6bada5e7bc53457d56fc92acac8cac29127acbcb):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-6bada5e7bc53457d56fc92acac8cac29127acbcb
Tests run against activerecord 7.2.1
8478 Total Tests Run
8473 tests passed
5 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18887368/activerecord/1740119705505/1740126596986)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18900479?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18900479?buildTab=artifacts#/activerecord) on master @ [c471a724f99a5add3ccefe07995251760d0e2212](https://github.com/cockroachdb/cockroach/commits/c471a724f99a5add3ccefe07995251760d0e2212):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c471a724f99a5add3ccefe07995251760d0e2212
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18900479/activerecord/1740206071893/1740213334068)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18906403?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18906403?buildTab=artifacts#/activerecord) on master @ [fa8342d50c1bdfbc307f4bca9bc0d34da4475258](https://github.com/cockroachdb/cockroach/commits/fa8342d50c1bdfbc307f4bca9bc0d34da4475258):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-fa8342d50c1bdfbc307f4bca9bc0d34da4475258
Tests run against activerecord 7.2.1
8478 Total Tests Run
8473 tests passed
5 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18906403/activerecord/1740292423614/1740299488258)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=artifacts#/activerecord) on master @ [cbb292c487ea7413772a5c3c349f93467264c190](https://github.com/cockroachdb/cockroach/commits/cbb292c487ea7413772a5c3c349f93467264c190):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-cbb292c487ea7413772a5c3c349f93467264c190
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18912532/activerecord/1740378635815/1740386464794)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=artifacts#/activerecord) on master @ [8bd38972db32f5e683ba6dc0eb599334919919e5](https://github.com/cockroachdb/cockroach/commits/8bd38972db32f5e683ba6dc0eb599334919919e5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8bd38972db32f5e683ba6dc0eb599334919919e5
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18923258/activerecord/1740465039488/1740473065045)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=artifacts#/activerecord) on master @ [32fe7ee0b575276bff803469006bba08737840cf](https://github.com/cockroachdb/cockroach/commits/32fe7ee0b575276bff803469006bba08737840cf):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-32fe7ee0b575276bff803469006bba08737840cf
Tests run against activerecord 7.2.1
8478 Total Tests Run
8474 tests passed
4 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18944241/activerecord/1740637876553/1740644834921)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18955318?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18955318?buildTab=artifacts#/activerecord) on master @ [56d996eda400f5d096ee921d5e95f6f4da124b73](https://github.com/cockroachdb/cockroach/commits/56d996eda400f5d096ee921d5e95f6f4da124b73):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-56d996eda400f5d096ee921d5e95f6f4da124b73
Tests run against activerecord 7.2.1
8478 Total Tests Run
8473 tests passed
5 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18955318/activerecord/1740724251465/1740731345682)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18965482?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18965482?buildTab=artifacts#/activerecord) on master @ [65de681fae64493f30647bd17dc77424e260c242](https://github.com/cockroachdb/cockroach/commits/65de681fae64493f30647bd17dc77424e260c242):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-65de681fae64493f30647bd17dc77424e260c242
Tests run against activerecord 7.2.1
8478 Total Tests Run
8475 tests passed
3 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18965482/activerecord/1740810568323/1740817506560)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18970888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18970888?buildTab=artifacts#/activerecord) on master @ [e527f6b34c009ed7b52777e854687fb5d6ce3378](https://github.com/cockroachdb/cockroach/commits/e527f6b34c009ed7b52777e854687fb5d6ce3378):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-e527f6b34c009ed7b52777e854687fb5d6ce3378
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/cpu_arch=arm64/run_1
```

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18970888/activerecord/1740896947099/1740907741733)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18975825?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18975825?buildTab=artifacts#/activerecord) on master @ [8c5af013aa725d4f441abeff282894ff22f04ba7](https://github.com/cockroachdb/cockroach/commits/8c5af013aa725d4f441abeff282894ff22f04ba7):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8c5af013aa725d4f441abeff282894ff22f04ba7
Tests run against activerecord 7.2.1
8478 Total Tests Run
8476 tests passed
2 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18975825/activerecord/1740983819724/1740992604075)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19001116?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19001116?buildTab=artifacts#/activerecord) on master @ [7fbd05617474b6a4885f662426c098e68323ad05](https://github.com/cockroachdb/cockroach/commits/7fbd05617474b6a4885f662426c098e68323ad05):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-7fbd05617474b6a4885f662426c098e68323ad05
Tests run against activerecord 7.2.1
8478 Total Tests Run
8473 tests passed
5 tests failed
60 tests skipped
14 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_circular_autosave_association_correctly_saves_multiple_records (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlArrayTest#test_uniqueness_validation (unexpected)
--- FAIL: PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19001116/activerecord/1741156145464/1741164801911)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #141974 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #140778 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.0-rc]
- #140579 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -27,8 +27,8 @@ var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\
 
 // WARNING: DO NOT MODIFY the name of the below constant/variable without approval from the docs team.
 // This is used by docs automation to produce a list of supported versions for ORM's.
-var supportedRailsVersion = "7.2.1"
-var activerecordAdapterVersion = "v7.2.0"
+var supportedRailsVersion = "8.0.1"
+var activerecordAdapterVersion = "v8.0.1"
 
 // This test runs activerecord's full test suite against a single cockroach node.
 

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -22,6 +22,7 @@ var activeRecordBlocklist = blocklist{
 }
 
 var activeRecordIgnoreList = blocklist{
+	`ActiveRecord::AdapterTestWithoutTransaction#test_create_with_query_cache`:                                                                                 "affected by autocommit_before_ddl",
 	`ActiveRecord::CockroachDBStructureDumpTest#test_structure_dump`:                                                                                           "flaky",
 	`ActiveRecord::ConnectionAdapters::ConnectionPoolThreadTest#test_checkout_fairness_by_group`:                                                               "flaky",
 	`ActiveRecord::ConnectionAdapters::PostgreSQLAdapterTest#test_translate_no_connection_exception_to_not_established`:                                        "pg_terminate_backend not implemented",
@@ -30,6 +31,11 @@ var activeRecordIgnoreList = blocklist{
 	`FixtureWithSetModelClassPrevailsOverNamingConventionTest#test_model_class_in_fixture_file_is_respected`:                                                   "flaky",
 	`InheritanceTest#test_eager_load_belongs_to_primary_key_quoting`:                                                                                           "flaky",
 	`InheritanceTest#test_eager_load_belongs_to_something_inherited`:                                                                                           "flaky",
+	`PostgresqlArrayTest#test_uniqueness_validation`:                                                                                                           "affected by autocommit_before_ddl",
+	`PostgresqlEnumTest#test_schema_dump_renamed_enum`:                                                                                                         "affected by autocommit_before_ddl",
+	`PostgresqlEnumTest#test_schema_dump_renamed_enum_with_to_option`:                                                                                          "affected by autocommit_before_ddl",
+	`PostgresqlInvertibleMigrationTest#test_migrate_revert_create_enum`:                                                                                        "affected by autocommit_before_ddl",
+	`PostgresqlInvertibleMigrationTest#test_migrate_revert_drop_enum`:                                                                                          "affected by autocommit_before_ddl",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update`: "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason`:                             "flaky",
 	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not`:            "flaky",

